### ISSUE TITLE
JAWS HTML support 

### ISSUE THREAD

**Issue by Rasubra8** [FreedomScientific/VFO-standards-support#125]
The JAWS commands mentioned in interaction column for navigating to block Quote is not Supported in the latest versions of JAWS. Request to modify the commands accordingly. 
### Summary

> Brief description of the issue. Please list any specific steps.
> 
JAWS support for various elements mentioned in the Table in the following page. 
https://freedomscientific.github.io/VFO-standards-support/html.html
JAWS commands for "block Quote" mentioned in the "interaction" column seems wrong. Pressing "Q" in The latest versions of JAWS takes the user to the main region of the page by default not to the Block Quote. This command was working by default in the older versions of JAWS. Hence I think the commands need to be modified accordingly.
### Expected result

> What did you expect to happen?

### Actual result

> What actually did happen?

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

**Comment by Menelion** [comment]
@stevefaulkner May I take this? I'd send a PR with the changes. there is no default quick nav key for blockquote, though, so I would reword this passage.

**Comment by jducrot** [comment]
Since JAWS 18, there is no more [keystroke listed to navigate a document using blockquote elements](https://support.freedomscientific.com/Content/Documents/Manuals/JAWS/Keystrokes.txt) so this is expected behavior.

**Comment by scottaohara** [comment]
this was actually a bug with the documentation provided in this repo.

I updated the keys regarding [navigating by block quotes in this commit](https://github.com/FreedomScientific/VFO-standards-support/commit/88413c15b6fc73fa58adbe5d245cc8bb48fbffe8), but was not aware of this issue when doing so. 

we should make sure to take another gander through the documentation hosted in this repo to make sure there are no other instances of updates that need to be made.
